1-js/02-first-steps/03-strict-mode/article.md

```diff
@@ -39,7 +39,7 @@ Only comments may appear above `"use strict"`.
 ````
 
 ```warn header="There's no way to cancel `use strict`"
-There is no directive like `"no use strict"` that reverts the engine to old behavior.
+There is no directive like `"no use strict"` that reverts the JavaScript engine to old behavior.
```

Comment:
to  the old
maybe?

Reply:
Hi,

Yes, you are right. I overlooked it.

Reply:
<img width="960" alt="Screenshot 2022-12-21 113411" src="https://user-images.githubusercontent.com/106430923/208872516-e4ab2bd3-1acf-4a9e-a12b-ca4f0758c53a.png">

Reply:
I have attached the discussion, it is in Romanian, but you can read the proposed version in English.
It seems I was very tired when I edited it, sorry.

The proposed version was :  ”There is no directive like "no use strict" that reverts the javascript engine used by browsers to the old behavior.” 

I removed ”used by browsers” but I forgot to put ”the”.

What do you think?